contact_map/contact_count.py

```diff
@@ -198,7 +198,7 @@ def _check_number_of_pixels(self, figure):
                    + " (width, height).")
             warnings.warn(msg, RuntimeWarning)
 
-    def plot(self, cmap='seismic', vmin=-1.0, vmax=1.0, with_colorbar=True,
+    def plot(self, cmap='seismic', diverging_cmap=None, with_colorbar=True,
```

Comment:
please also update the docstring for this function

Reply:
Should be fixed in newest commit.